.github/workflows/multi-gpu-e2e.yml

```
@@ -26,7 +31,19 @@ env:
 
 jobs:
   test-axolotl-multigpu:
-    if: ${{ ! contains(github.event.head_commit.message, '[skip e2e]') && github.repository_owner == 'axolotl-ai-cloud' && (github.event_name != 'pull_request' || !github.event.pull_request.draft) }}
+    if: >
```

actionlint (1.7.12)

[error] if: condition "${{\n  !contains(github.event.head_commit.message, '[skip e2e]') &&\n  github.repository_owner == 'axolotl-ai-cloud' &&\n  (\n    github.event_name != 'pull_request' ||\n    (\n      !github.event.pull_request.draft &&\n      contains(github.event.pull_request.labels.*.name, 'run-gpu-tests') &&\n      (github.event.action != 'labeled' || github.event.label.name == 'run-gpu-tests')\n    )\n  )\n}}\n" is always evaluated to true because extra characters are around ${{ }}

(if-cond)